scripts/core/superdesk-api.d.ts

```diff
@@ -257,6 +257,12 @@ declare module 'superdesk-api' {
         order: number,
     }
 
+    export interface IDimension {
```

Comment:
This metadata is specific to SAMS, and `IAssetItem`, so it would be better off going into `scripts/extensions/sams/src/interfaces.ts`.

The interface also has the following schemas (as based on schema from SAMS https://github.com/superdesk/sams/blob/develop/src/clients/python/sams_client/schemas/assets.py#L48):
```typescript
export interface IAssetRenditionArgs {
    width: number;
    height: number;
    keep_proportions: boolean;
}

export interface IAssetRendition {
    _media_id: string;
    width: number; // actual width
    height: number; // actual height
    params: IAssetRenditionArgs; // arguments used when the rendition was generated
    versioncreated: string;
    filename: string;
    length: number;
}

export interface IAssetItem extends IVersionInformation {
    ....
    renditions: Array<IAssetRendition>;
}
```